### PR TITLE
Infrastructure: Fix color contrast failure in HTML source display

### PR DESCRIPTION
**Pull request by evmiguel**
This PR addresses #2815. I upgraded highlight.js to get the latest features, as we were behind a few major versions.
___
[WAI Preview Link](https://deploy-preview-301--aria-practices.netlify.app/ARIA/apg) _(Last built on Thu, 29 Feb 2024 15:49:24 GMT)._

### CODE REVIEW

**Review by alflennik — APPROVED** [review]
Nice fix! I didn't understand 100% about the change from the "<br>" to "\n" so in addition to checking this example I went through about half of the examples just verifying that they all look right, as well as reading them with a screen reader. Everything seems perfect. Happy to approve.

**Comment by evmiguel** [comment]
@alflennik The change from `<br>` to `\n` was because highlightjs stopped supporting break tags. See https://github.com/highlightjs/highlight.js/issues/3285#issuecomment-885995008

**Review by howard-e — APPROVED** [review]
@evmiguel thanks for demoing this recently! And can confirm this addresses the reported issues from #2815:
* [Date Picker Dialog Example](https://deploy-preview-301--aria-practices.netlify.app/aria/apg/patterns/combobox/examples/combobox-datepicker/#sc1_label)
* [Example Disclosure Navigation Menu with Top-Level Links](https://deploy-preview-301--aria-practices.netlify.app/aria/apg/patterns/disclosure/examples/disclosure-navigation-hybrid/#sc1_label)

**Comment by howard-e** [comment]
@a11ydoer could you also confirm this resolves the issue as expected?

File: content/shared/js/highlight.pack.js [hunk 1]
```diff
@@ -1,2 +1,337 @@
...
\ No newline at end of file
+/*!
+  Highlight.js v11.9.0 (git: b7ec4bfafc)
+  (c) 2006-2023 undefined and other contributors
```
Comment (nschonni) [inline content/shared/js/highlight.pack.js:3]: Looks like the license export might have glitched

Comment (evmiguel) [reply content/shared/js/highlight.pack.js:3]: @nschonni thanks for catching this. I just downloaded the file for consumption. Is there a good way to fix this?

Comment (nschonni) [reply content/shared/js/highlight.pack.js:3]: Not sure, maybe it was a temporary glitch on the site generator, assuming that's how you grabbed it

Comment (howard-e) [reply content/shared/js/highlight.pack.js:3]: Yep confirming I'm also getting `undefined` from the exporter at https://highlightjs.org/download. We could make an assumption about that name based on the [repository's LICENSE](https://github.com/highlightjs/highlight.js/blob/main/LICENSE), though we shouldn't add include it.

Seems like something we can make an issue about on their repository and update this file when that's fixed.

**Review by a11ydoer — APPROVED** [review]
Thanks for fixing the inconsistent color coding issues while you are also updating the package. They look great.

**Pull request merged**